demos/netgen/NetGenPoisson.rst

```diff
@@ -0,0 +1,89 @@
+We import the needed library from FireDrake and NetGen.
+
+.. code:: ipython3
+
+    from firedrake import *
+    import firedrake.cython.dmcommon as dmcommon
+    from firedrake import mesh
+    from ngsolve import Mesh,Draw
+    from netgen.geom2d import CSG2d, Circle, Rectangle
+
+we define a geometry using NetGen constructive geometry tools.
+
+.. code:: ipython3
+
+    #EXAMPLE OF Constructive Solid Geometry
```

Comment:
Prefer rst text outside code blocks to comments. You can have as many code blocks as are needed to explain what is going on.